docs/components/01-header-nav.mdx

```diff
@@ -0,0 +1,45 @@
+# Header Nav
+
+<Description
+  extendsInfo={[
+    {
+      label: 'NavLink',
+      link: 'https://v6.mantine.dev/core/nav-link/',
+    },
+    {
+      label: 'SidebarMenu',
+      link: 'https://smile-sa.github.io/react-front-kit/?path=/docs/3-custom-components-sidebarmenu--docs',
+    },
+  ]}
+  importExample="import { HeaderNav } from '@smile/react-front-kit';"
+  packageInfo={{
+    label: '@smile/react-front-kit',
+    link: 'https://www.npmjs.com/package/@smile/react-front-kit',
+  }}
+  sourceInfo={{
+    link: 'https://github.com/Smile-SA/react-front-kit/blob/main/packages/react-front-kit/src/Components/HeaderNav/HeaderNav.tsx',
+  }}
+>
+  Render a navigation for header
```

Comment:
ça aide pas du tout comme description, faut que ça soit un peu plus parlant, comme "Renders a list of menus as either a horizontal list of NavLink components or a vertical SidebarMenu in mobile mode".
La description doit expliquer ce que "fait" le component pour un developpeur, son intention.